crates/parser/src/codegen.rs

```diff
@@ -66,10 +66,7 @@ mod tests {
 
         debug!("selected node: {:#?}", node_graph[node_index]);
 
-        assert!(node_graph[node_index]
-            .properties
-            .iter()
-            .all(|p| { expected.contains(p) }));
+        assert_eq!(node_graph[node_index].properties, expected);
```

Comment:
I don't have a strong opinion here but I find it was easier to debug before #62 for 2 reasons:
- strict assertion - so we cannot have unexpected "extra" nodes
- better assertion message of what is missing in the output

cc @psteinroe

Reply:
Can you add a comment there that even though we test for strict equality of the two vectors the order of the properties does not have to match the order of the tokens in the string?

Reply:
either this, or I just tweak the assertion message - happy to revert if requested